www/grpc/gen/go/blockchain.pb.jgw.go

```diff
@@ -0,0 +1,121 @@
+// Code generated by protoc-gen-jrpc-gateway. DO NOT EDIT.
+// source: blockchain.proto
+
+/*
+Package pactus is a reverse proxy.
+
+It translates gRPC into JSON-RPC 2.0
+*/
+package pactus
+
+import (
+	"context"
+	"encoding/json"
+
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type BlockchainJsonRpcService struct {
+	client BlockchainClient
+}
+
+func NewBlockchainJsonRpcService(client BlockchainClient) BlockchainJsonRpcService {
+	return BlockchainJsonRpcService{
+		client: client,
+	}
+}
+
```

Comment:
You need to add a function here like:

```go
// RegisterBlockchainMethods registers the JSON-RPC methods for service Blockchain to "mux".
// The handlers forward requests to the grpc endpoint over "conn".
func RegisterTransactionHandler(ctx context.Context, mux *jgw.ServerMux, conn *grpc.ClientConn) error {
	return mux.RegisterMethods(ctx, BlockchainMethods(NewTransactionClient(conn)))
}

```

Reply:
You can give a type to: `func(ctx context.Context, params json.RawMessage) (interface{}, error)`